crates/cow-amm/src/lib.rs

```diff
@@ -5,3 +5,16 @@ mod maintainers;
 mod registry;
 
 pub use {amm::Amm, contracts::CowAmmLegacyHelper as Helper, registry::Registry};
+
+#[derive(prometheus_metric_storage::MetricStorage)]
+pub(crate) struct Metrics {
+    /// Timing of db queries.
```

Comment:
Is timing here when they happen or how long they take? I think this could be slightly clearer